test/NuGet.Core.Tests/NuGet.Frameworks.Test/CompatibilityTests.cs

```diff
@@ -599,6 +599,19 @@ public class CompatibilityTests
         [InlineData("netcoreapp1.0", "net451", false)]
         [InlineData("netcoreapp1.0", "net4", false)]
         [InlineData("netcoreapp1.0", "win8", false)]
+
+        // UAP 10.0.15064.0 is compatible with netstandard2.0 
+        // Reference - https://devdiv.visualstudio.com/DevDiv/_queries?id=469873 
+        [InlineData("uap10.0.15064.0", "netstandard2.0", true)]
+        [InlineData("uap10.0.15064.0", "dotnet", true)]
+        [InlineData("uap10.0.15064.0", "netcoreapp2.0", false)]
+        [InlineData("uap10.0.15064.0", "netcoreapp1.0", false)]
+        [InlineData("uap10.0.15064.0", "net45", false)]
+        [InlineData("uap10.0.15064.0", "net46", false)]
+        [InlineData("uap10.0.15064.0", "netstandardapp1.5", false)]
+        [InlineData("uap10.0.15064.0", "netstandardapp1.5", false)]
+        [InlineData("uap10.0.15064.0", "netstandardapp1.5", false)]
+        [InlineData("uap10.0.15064.0", "netstandardapp1.5", false)]
```

Comment:
Looks like these last ones weren't filled in.

This is really just testing the compat mappings system itself, but you could add these boundary cases:
`[InlineData("uap10.0.15064.0", "netstandard2.1", false)]`
`[InlineData("uap10.0.15063.0", "netstandard2.0", false)]`
`[InlineData("uap10.0.15064.0", "netstandard1.9", true)]`